src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java

```diff
@@ -621,9 +769,17 @@ public void setPort(int port) {
     public int getPort() {
         return port;
     }
+    
+    public void setHostAddresses(String hostAddresses) {
```

Comment:
Honestly, I've never really liked the name "host" / "port", because it doesn't indicate what is the "host" of  (source/destination?).  How about we do this:

```
/*
 * Add the given comma separated list of destination addresses ("hostname:port")
 * to the list of possible destinations to which to send logs.
 * Hosts will be tried in the order in which they are added.
 * Logs will be sent to the first one which accepts the connection.
 */
void addDestination(String destination)
```

And then the logback config be this:
```
<appender....>
    <destination>hostname1:port1, hostname2:port2</destination>
</appender>
```

OR this:
```
<appender....>
    <destination>hostname1:port1<destination>
    <destination>hostname2:port2</destination>
</appender>
```

And then we can deprecate the host/port fields.